### DISCUSSION TITLE
Welcome to 4H-makerspace Discussions!

### DISCUSSION THREAD

**Discussion by JackBoreczky** [JackBoreczky/4H-makerspace#11] [General]
## 👋 Welcome!
Discussions are here! Another fun Github Feature to play with. I did make the repository public to access this feature but it won't change how you interact with the repo and if you have a fork of the repo, it'll stay private.

While I'm here, we have a meeting next week on Wednesday the 27th! Mark your calendars. I'll be trying out a Python activity that I don't have fully completed yet but it will hopefully be a nice break from the complexity of Github.

**Comment by bitbloxhub** [comment]
**WARNING**: merging forks may not work. make a new branch and pr that instead.
